src/pages/settings/Profile/ProfilePage.js

```diff
@@ -80,23 +81,34 @@ class ProfilePage extends Component {
             selectedTimezone: lodashGet(props.myPersonalDetails.timezone, 'selected', CONST.DEFAULT_TIME_ZONE.selected),
             isAutomaticTimezone: lodashGet(props.myPersonalDetails.timezone, 'automatic', CONST.DEFAULT_TIME_ZONE.automatic),
             logins: this.getLogins(props.user.loginList),
+            avatarImage: null,
+            avatarPreviewURL: lodashGet(this.props.myPersonalDetails, 'avatar', OptionsListUtils.getDefaultAvatar(this.props.myPersonalDetails.login)),
+            isAvatarUpdated: false,
         };
+
         this.getLogins = this.getLogins.bind(this);
         this.setAutomaticTimezone = this.setAutomaticTimezone.bind(this);
         this.updatePersonalDetails = this.updatePersonalDetails.bind(this);
         this.validateInputs = this.validateInputs.bind(this);
```

Comment:
I think you forgot to bind `setAvatar()` and `removeAvatar()`

Reply:
In my previous PRs, I was getting feedback to not add unnecessary bindings.

Reply:
But in this case we need it so that the function can access `this`

Reply:
I was referring to this guide: https://github.com/Expensify/App/blob/main/STYLE.md#binding-methods, specifically `Binding all methods in the constructor is unnecessary.`

Reply:
Nevertheless, I've added now. Please check

Reply:
All done here, I'll test on all platforms and approve the PR.